src/pg/sql/clauses.js

```diff
@@ -24,16 +24,20 @@ export const lookup = (prop) => {
   const [prefix, ...suffix] = encodePath(prop);
   return suffix.length
     ? // @ts-ignore sql-template-tag typedef bug
-      sql`"${raw(prefix)}" #> ${suffix}`
+      sql`CASE WHEN ("${raw(
+        prefix,
+      )}" #> ${suffix})::text = 'null' THEN '0'::JSONB ELSE "${raw(
+        prefix,
+      )}" #> ${suffix} END`
```

Comment:
Don't change `lookup` as it is used in other places as well.

Maybe create a new function, lookupNumeric:

```js
export const lookupNumeric = (prop) => {
      const lookupSql = lookup(props)
      sql`CASE WHEN ${lookupSql} = 'null'::jsonb or ${lookupSql}) IS NULL
          THEN 0 ELSE ${lookupSql}::numeric END`
```

The aggSql functions can then be:

```
$sum: (prop) => sql`sum(${lookupNumeric(prop)})`,
```